samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -21,29 +37,41 @@ The sample demonstrates the following features:
 
 This sample configures a single device as a transmitter of its **BUTTON1** state.
 The transmitting and receiving devices toggle **LED1** synchronously with the accuracy of a few microseconds.
-When the :kconfig:option:`LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE` Kconfig option is enabled, **LED2** is toggled upon sending or receiving data.
+When the :ref:`CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE <CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE>` Kconfig option is enabled, **LED2** is toggled upon sending or receiving data.
 This allows you to measure the minimal end-to-end latency.
 
 .. note::
-  This sample requires less hardware resources when it is run on an nRF54L Series device compared to the nRF52 or nRF53 Series devices.
-  On an nRF54L Series device, only one GRTC channel and PPI channel is needed to set up accurate toggling of an LED.
-  On nRF52 and nRF53 Series devices, you also need one RTC peripheral, one TIMER peripheral, one EGU channel, three PPI channels, and one PPI group.
+   This sample requires less hardware resources when it is run on an nRF54L Series device compared to the nRF52 or nRF53 Series devices.
+   On an nRF54L Series device, only one GRTC channel and PPI channel is needed to set up accurate toggling of an LED.
+   On nRF52 and nRF53 Series devices, you also need one RTC peripheral, one TIMER peripheral, one EGU channel, three PPI channels, and one PPI group.
 
-Requirements
-************
+Configuration
+*************
 
-The sample supports the following development kits:
+|config|
 
-.. table-from-sample-yaml::
+Configuration options
+=====================
 
-The sample also supports other development kits running SoftDevice Controller variants that support isochronous channels (see :ref:`SoftDevice Controller feature support <softdevice_controller>`).
-To demonstrate time synchronization between the transmitter and a receiver, you need two development kits.
-To demonstrate time synchronization between the transmitter and multiple receivers, you need at least three development kits.
-You can use any combination of the development kits mentioned above, mixing different development kits.
+Check and configure the following Kconfig options:
 
-Additionally, the sample requires a connection to a computer with a serial terminal for each development kit in use.
+.. _CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE
 
-To observe the toggling of the LEDs, use a logic analyzer or an oscilloscope.
+CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE
+   This configuration option enables immediate toggling of **LED2** when isochronous data is sent or received.
+   It allows for measurement of the mimimum end-to-end latency.
+
+.. _CONFIG_SDU_INTERVAL_US
+
+CONFIG_SDU_INTERVAL_US
+   This configuration option determines how often the button value is transmitted to the receiving devices.
+
+.. _CONFIG_TIMED_LED_PRESENTATION_DELAY_US
+
+CONFIG_TIMED_LED_PRESENTATION_DELAY_US
```

Comment:
```suggestion
CONFIG_TIMED_LED_PRESENTATION_DELAY_US:
```

Reply:
Suggested fix on wrong line. Line 69 needs a colon at the end.